addons/mail/static/src/models/rtc_call_participant_card/rtc_call_participant_card.js

```diff
@@ -51,11 +58,12 @@ function factory(dependencies) {
         }
 
         /**
+         * Handled by the popover component.
+         *
          * @param {MouseEvent} ev
          */
-        async onClickVideo(ev) {
-            ev.stopPropagation();
-            this.messaging.toggleFocusedRtcSession(this.rtcSession.id);
+        async onClickVolumeAnchor(ev) {
+            markEventHandled(ev, 'CallParticipantCard.clickVolumeAnchor');
```

Comment:
This is a bit annoying to have such marking of handled event with not clear reason of action that require such marking. Here we have to read comment and trust the handling of popover that doesn't integrate well with models.

It would have been best to use `PopoverView` to solve this issue, but this is fix in stable so not a good idea to make bigger change that absolutely necessary.